hax/helper/configure.py

```diff
@@ -51,20 +52,32 @@ def _setup_logging():
               help='Target folder where Hare-related configuration will '
               'be written to.',
               show_default=True)
+@click.option('--uuid',
+              type=str,
+              help='UUID to be used',
+              show_default=True)
```

Comment:
[Minor] There is no default value anyway. Probably `show_default` makes no sense.